### PR TITLE
docs: use "Data Model" instead of "Data Schema"

### PR DESCRIPTION
**Pull request by hassankhan**

### CODE REVIEW

**Comment by keydunov** [comment]
@igorlukanin Can you please help to review this one? Thank you!

File: docs/content/Caching/Overview.mdx [hunk 1]
```diff
@@ -49,8 +49,8 @@ more about read-only support and pre-aggregation build strategies.
 
 </InfoBox>
 
-Pre-aggregations are defined in the data schema. You can learn more about
-defining pre-aggregations in [schema reference][ref-schema-ref-preaggs].
+Pre-aggregations are defined in the data model. You can learn more about
+defining pre-aggregations in [data modelling reference][ref-schema-ref-preaggs].
```
Comment (igorlukanin) [inline docs/content/Caching/Overview.mdx:53]:
```suggestion
defining pre-aggregations in [data modeling reference][ref-schema-ref-preaggs].
```
Sorry 🇺🇸🗽🙂

File: docs/content/Caching/Using-Pre-Aggregations.mdx [hunk 2]
```diff
@@ -7,7 +7,8 @@ menuOrder: 3
 
 Pre-aggregations is a powerful way to speed up your Cube queries. There are many
 configuration options to consider. Please make sure to also check [the
-Pre-Aggregations reference in the data schema section][ref-schema-ref-preaggs].
+Pre-Aggregations reference in the data modelling
```
Comment (igorlukanin) [inline docs/content/Caching/Using-Pre-Aggregations.mdx:10]:
```suggestion
Pre-Aggregations reference in the data modeling
```

File: docs/content/Caching/Overview.mdx (hunk 1 above)
Comment (hassankhan) [reply docs/content/Caching/Overview.mdx:53]: Ahh good catch, my bad 😀

File: docs/content/Deployment/Overview.mdx [hunk 3]
```diff
@@ -57,14 +57,14 @@ requests between multiple API instances.
 
 The [Cube Docker image][dh-cubejs] is used for API Instance.
 
-API instance needs to be configured via environment variables, cube.js file and
-has access to the data schema files.
+API instance needs to be configured via environment variables, the `cube.js`
+configuration file and has access to the data model files.
```
Comment (igorlukanin) [inline docs/content/Deployment/Overview.mdx:61]: Sounds weird. Who/what has access to the data model files? Maybe you can quickly untangle this now 🙂

File: docs/content/Deployment/Overview.mdx [hunk 4]
```diff
@@ -57,14 +57,14 @@ requests between multiple API instances.
...
 
 ## Refresh Worker
 
 A Refresh Worker updates pre-aggregations and invalidates the in-memory cache in
-the background. They also keep the refresh keys up-to-date for all defined
-schemas and pre-aggregations. Please note that the in-memory cache is just
+the background. They also keep the refresh keys up-to-date for all defined data
```
Comment (igorlukanin) [inline docs/content/Deployment/Overview.mdx:66]:
```suggestion
the background. They also keep the refresh keys up-to-date for all data
```

File: docs/content/Deployment/Production-Checklist.mdx [hunk 5]
```diff
@@ -97,37 +97,45 @@ deployment's health and be alerted to any issues.
...
 
-There's no one-size-fits-all when it comes to sizing Cube cluster, and its resources.
-Resources required by Cube depend a lot on the amount of traffic Cube needs to serve and the amount of data it needs to process.
-The following sizing estimates are based on default settings and are very generic, which may not fit your Cube use case, so you should always tweak resources based on consumption patterns you see.
+There's no one-size-fits-all when it comes to sizing Cube cluster, and its
+resources. Resources required by Cube depend a lot on the amount of traffic Cube
+needs to serve and the amount of data it needs to process. The following sizing
+estimates are based on default settings and are very generic, which may not fit
+your Cube use case, so you should always tweak resources based on consumption
+patterns you see.
```
Comment (igorlukanin) [inline docs/content/Deployment/Production-Checklist.mdx:105]:
```suggestion
There's no one-size-fits-all when it comes to sizing Cube cluster and its
resources. Resources required by Cube depend a lot on the amount of traffic Cube
needs to serve and the amount of data it needs to process. The following sizing
estimates are based on default settings and are very generic, which may not fit
your Cube use case, so you should always tweak resources based on consumption
patterns you see.
```

File: docs/content/Examples-Tutorials-Recipes/Examples.mdx [hunk 6]
```diff
@@ -41,13 +41,13 @@ The following tutorials cover advanced concepts of Cube:
...
+| [Compare date range](https://cube.dev/docs/query-format#time-dimensions-format)         | [Comparing data over different time periods](https://cube.dev/blog/comparing-data-over-different-time-periods/)   | [Demo](https://compare-date-range-demo.cube.dev)  |
+| [Data blending](https://cube.dev/docs/recipes/data-blending)                            | [Introducing data blending API](https://cube.dev/blog/introducing-data-blending-api/)                             | [Demo](https://data-blending-demo.cube.dev)       |
+| [Real-time data fetch](https://cube.dev/docs/real-time-data-fetch)                      | [Real-time dashboard guide](https://real-time-dashboard.cube.dev)                                                 | [Demo](https://real-time-dashboard-demo.cube.dev) |
+| [Dynamic model creation](https://cube.dev/docs/dynamic-schema-creation)                 | [Using asyncModule to generate schemas](https://github.com/cube-js/cube/tree/master/examples/async-module-simple) | —                                                 |
```
Comment (igorlukanin) [inline docs/content/Examples-Tutorials-Recipes/Examples.mdx:50]:
```suggestion
| [Dynamic data model](https://cube.dev/docs/dynamic-schema-creation)                 | [Using asyncModule to generate schemas](https://github.com/cube-js/cube/tree/master/examples/async-module-simple) | —                                                 |
```

File: docs/content/Examples-Tutorials-Recipes/Recipes/schema-generation.mdx [hunk 7]
```diff
@@ -9,7 +9,7 @@ redirect_from:
 ---
 
 Cube Schema is Javascript code, which means the full power of this language can
-be used to configure your schema definitions. In this guide we generate several
+be used to configure your data model definitions. In this guide we generate several
 measure definitions based on an array of strings.
```
Comment (igorlukanin) [inline docs/content/Examples-Tutorials-Recipes/Recipes/schema-generation.mdx:13]: Maybe this whole thing can be rewritten as follows:

> Cube supports two ways to define data model files: with YAML or JavaScript syntax. If you opt for JavaScript syntax, you can use the full power of this programming language to configure your data model. In this guide...

File: docs/content/Schema/Fundamentals/Additional-Concepts.mdx [hunk 8]
```diff
@@ -232,7 +232,7 @@ cubes:
         type: string
         primary_key: true
       - name: deals_amount
-        sql: "{Deals.amount}"
+        sql: '{Deals.amount}'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Additional-Concepts.mdx:235]: Same. @hassankhan Can you maybe tune prettier?

File: docs/content/Schema/Fundamentals/Additional-Concepts.mdx [hunk 9]
```diff
@@ -284,7 +284,7 @@ cubes:
   - name: SalesManagers
     dimensions:
       - name: deals_amount
-        sql: "SUM({Deals}.amount)" # Doesn't work, because `amount` is not a measure on `Deals`
+        sql: 'SUM({Deals}.amount)' # Doesn't work, because `amount` is not a measure on `Deals`
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Additional-Concepts.mdx:287]: Same.

File: docs/content/Schema/Fundamentals/Additional-Concepts.mdx [hunk 10]
```diff
@@ -326,15 +326,15 @@ cubes:
   - name: SalesManagers
     measures:
       - name: average_deals_amount
-        sql: "{CUBE.deals_amount}"
+        sql: '{CUBE.deals_amount}'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Additional-Concepts.mdx:329]: Same.

File: docs/content/Schema/Fundamentals/Additional-Concepts.mdx [hunk 11]
```diff
@@ -326,15 +326,15 @@ cubes:
...
         type: avg
     dimensions:
       - name: id
         sql: id
         type: string
         primary_key: true
       - name: deals_amount
-        sql: "{Deals.amount}"
+        sql: '{Deals.amount}'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Additional-Concepts.mdx:337]: Same.

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx [hunk 12]
```diff
@@ -345,7 +345,7 @@ cubes:
   - name: PostTopics
     dimensions:
       - name: id
-        sql: "CONCAT({CUBE}.post_id, {CUBE}.topic_id)"
+        sql: 'CONCAT({CUBE}.post_id, {CUBE}.topic_id)'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Working-with-Joins.mdx:348]: Same.

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx [hunk 13]
```diff
@@ -568,7 +568,9 @@ cubes:
     joins:
       - name: Campaigns
         relationship: many_to_one
-        sql: "{CUBE}.campaign_id = {Campaigns.id} AND {CUBE}.customer_name = {Campaigns.customer_name}"
+        sql:
+          '{CUBE}.campaign_id = {Campaigns.id} AND {CUBE}.customer_name =
+          {Campaigns.customer_name}'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Working-with-Joins.mdx:573]: Same. Also, single-line looked better.

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx [hunk 14]
```diff
@@ -588,7 +590,9 @@ cubes:
     joins:
       - name: Transactions
         relationship: one_to_many
-        sql: "{CUBE}.customer_name = {Transactions.customer_name} AND {CUBE}.campaign_id = {Transactions.campaign_id}"
+        sql:
+          '{CUBE}.customer_name = {Transactions.customer_name} AND
+          {CUBE}.campaign_id = {Transactions.campaign_id}'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Working-with-Joins.mdx:595]: Same.

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx [hunk 15]
```diff
@@ -902,7 +906,7 @@ views:
     measures:
       - name: count
         # Note the nested path to Orders.count
-        sql: "{Users.Orders.count}"
+        sql: '{Users.Orders.count}'
```
Comment (igorlukanin) [inline docs/content/Schema/Fundamentals/Working-with-Joins.mdx:909]: Same.

File: docs/content/Schema/Getting-Started.mdx [hunk 16]
```diff
@@ -247,7 +248,7 @@ cubes:
         filters:
           - sql: "{CUBE}.paying = 'true'"
       - name: paying_percentage
-        sql: "100.0 * {paying_count} / {count}"
+        sql: '100.0 * {paying_count} / {count}'
```
Comment (igorlukanin) [inline docs/content/Schema/Getting-Started.mdx:251]: Same.

File: docs/content/Schema/Reference/cube.mdx [hunk 17]
```diff
@@ -339,15 +338,15 @@ cubes:
     measures:
       - name: double_count
         type: number
-        sql: "{count} * 2"
+        sql: '{count} * 2'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/cube.mdx:341]: Same.

File: docs/content/Schema/Reference/cube.mdx [hunk 18]
```diff
@@ -996,7 +995,7 @@ cubes:
       - name: createdAtConverted
         # do not use in timeDimensions query property
         type: time
-        sql: "{SQL_UTILS.convertTz(`created_at`)}"
+        sql: '{SQL_UTILS.convertTz(`created_at`)}'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/cube.mdx:998]: Same.

File: docs/content/Schema/Reference/pre-aggregations.mdx [hunk 19]
```diff
@@ -442,7 +442,7 @@ scenarios where real-time data is required.
 
 [Lambda pre-aggregations][ref-caching-lambda-preaggs] can be used to combine
 data from a data source and a pre-aggregation, or even from multiple
-pre-aggregations across different schema-compatible cubes.
+pre-aggregations across different data model-compatible cubes.
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/pre-aggregations.mdx:445]: Just saying: I have no idea what "data model-compatible cubes" are.

File: docs/content/Schema/Reference/types-and-formats.mdx [hunk 20]
```diff
@@ -72,7 +72,7 @@ cubes:
   - name: Orders
     measures:
       - name: last_order
-        sql: "MAX(created_at)"
+        sql: 'MAX(created_at)'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/types-and-formats.mdx:75]: Same.

File: docs/content/Schema/Reference/types-and-formats.mdx [hunk 21]
```diff
@@ -167,7 +168,7 @@ cubes:
   - name: Orders
     measures:
       - name: ratio
-        sql: "SUM({CUBE}.amount) / count(*)"
+        sql: 'SUM({CUBE}.amount) / count(*)'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/types-and-formats.mdx:171]: Same.

File: docs/content/Schema/Reference/types-and-formats.mdx [hunk 22]
```diff
@@ -311,7 +312,7 @@ cubes:
   - name: Orders
     measures:
       - name: revenue
-        sql: "{chargesAmount}"
+        sql: '{chargesAmount}'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/types-and-formats.mdx:315]: Same.

File: docs/content/Schema/Reference/types-and-formats.mdx [hunk 23]
```diff
@@ -511,7 +512,7 @@ cubes:
   - name: Orders
     measures:
       - name: purchase_conversion
-        sql: "{purchase}/{checkout}*100.0"
+        sql: '{purchase}/{checkout}*100.0'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/types-and-formats.mdx:515]: Same.

File: docs/content/Schema/Reference/types-and-formats.mdx [hunk 24]
```diff
@@ -706,9 +708,9 @@ cubes:
       - name: location
         type: geo
         latitude:
-          sql: "{CUBE}.latitude"
+          sql: '{CUBE}.latitude'
         longitude:
-          sql: "{CUBE}.longitude"
+          sql: '{CUBE}.longitude'
```
Comment (igorlukanin) [inline docs/content/Schema/Reference/types-and-formats.mdx:713]: Same.

**Review by igorlukanin — APPROVED** [review]
Huge work, @hassankhan. Minor nits but otherwise LGTM!

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx (hunk 13 above)
Comment (hassankhan) [reply docs/content/Schema/Fundamentals/Working-with-Joins.mdx:573]: Prettier being annoying here again

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx (hunk 14 above)
Comment (hassankhan) [reply docs/content/Schema/Fundamentals/Working-with-Joins.mdx:595]: Prettier being annoying here again

File: docs/content/Schema/Reference/pre-aggregations.mdx (hunk 19 above)
Comment (hassankhan) [reply docs/content/Schema/Reference/pre-aggregations.mdx]: Changed to:

> or even from multiple pre-aggregations across different cubes that share the same dimensions
and measures.

**Pull request merged**